mirgecom/flux.py

```diff
@@ -35,6 +44,40 @@
 import numpy as np  # noqa
 
 
+def num_flux_lfr(f_minus, f_plus, q_minus, q_plus, lam, **kwargs):
+    """Lax-Friedrichs/Rusanov low-level numerical flux."""
+    return (f_minus + f_plus - lam*(q_plus - q_minus))/2
+
+
+def num_flux_central(f_minus, f_plus, **kwargs):
+    """Central low-level numerical flux."""
+    return (f_plus + f_minus)/2
+
+
+def hll_flux_driver(state_pair, physical_flux_func,
```

Comment:
How come this is not called `num_flux_hll`? How come its interface is inconsistent with the other fluxes?

Reply:
Fixed I think.

Reply:
Resolving this as we've addressed in #648